WalletWasabi/Wallets/BlockProvider/ISourceData.cs

```diff
@@ -60,7 +60,7 @@ public record EmptySourceData(Source Source) : ISourceData
 /// Number of connected peers at the moment when we downloaded the bitcoin block.
 /// The number of connected peers can change at any moment.
 /// </param>
-public record P2pSourceData(P2pSourceDataStatusCode StatusCode, Node? Node, uint ConnectedNodes) : ISourceData
+public record P2pSourceData(P2pSourceDataStatusCode StatusCode, TimeSpan Duration, Node? Node, uint ConnectedNodes) : ISourceData
```

Comment:
f100aac: Comment
4270482: Duration. I kept the timer before node discovery to keep accounting how long we took in case we are failing to find a `Node`, but I restarted the `Stopwatch` afterwards if a `Node` was found.